app/models/api_namespace.rb

```diff
@@ -105,11 +108,11 @@ def duplicate_api_namespace(duplicate_associations: false)
             new_api_form.save!
           end
     
-          # Duplicate ApiClients
-          self.api_clients.each do |api_client|
-            new_api_client = api_client.dup
-            new_api_client.api_namespace = new_api_namespace
-            new_api_client.save!
+          # Duplicate ApiNamespaceKeys
```

Comment:
Since we have introduced ApiNamespaceKey and ApiKey in lieu of ApiClient, I think we will need to create ApiKeys as well in similar way. Would you please confirm if ApiKeys are also duplicated _[or  if it needs to be duplicated]_?